tools/packaging/kernel/build-kernel.sh

```diff
@@ -132,6 +132,23 @@ get_tdx_kernel() {
 	tar --strip-components=1 -xf ${kernel_tarball} -C ${kernel_path}
 }
 
+get_sev_kernel() {
+	local version="${1}"
+	local kernel_path=${2}
+
+	mkdir -p ${kernel_path}
+
+	kernel_url=$(get_from_kata_deps "assets.kernel.sev.url")
+	kernel_tarball="${version}.tar.gz"
+
+	if [ ! -f "${kernel_tarball}" ]; then
+	   curl --fail -OL "${kernel_url}${kernel_tarball}"
+	fi
+	
+	mkdir -p ${kernel_path}
+	tar --strip-components=1 -xf ${kernel_tarball} -C ${kernel_path}
```

Comment:
I suspect this is based on `get_tdx_kernel()` which has the same limitation but I'd prefer that all downloads are checksumed, as we do in the default `get_kernel()`.

Reply:
`efi_secret` should be part of the mainline kernel when 5.19 is released in about 8-9 weeks. So maybe for now it's good enough without the checksum, and after 5.19 is released we can just download an official checksummed file from kernel.org.